### PR TITLE
Myrkiti Farm -> Aqueduct Bridge Area - scrnshot edit

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -2597,8 +2597,8 @@ QUEST_LV_0200_20150317_002596	There is a suspicious shadow in Spring Light Woods
 QUEST_LV_0200_20150317_002597	Chafer was hiding in the altar. Defeat the chafer.
 QUEST_LV_0200_20150317_002598	Check the destroyed seal tower
 QUEST_LV_0200_20150317_002599	Someone destroyed the tower. Check its status.
-QUEST_LV_0200_20150317_002600	Defeat the Mantiken
-QUEST_LV_0200_20150317_002601	Seems like the Mantiken destroyed the tower. Defeat it.
+QUEST_LV_0200_20150317_002600	$Defeat the Mentiken
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Defeat the Mentiken -> Defeat Manticen

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -2597,8 +2597,8 @@ QUEST_LV_0200_20150317_002596	There is a suspicious shadow in Spring Light Woods
...
+QUEST_LV_0200_20150317_002601	$Seems like the Mentiken destroyed the tower. Defeat it.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Seems like the Mentiken destroyed the tower. -> Seems like Manticen destroyed the tower.

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -4391,23 +4391,23 @@ QUEST_LV_0200_20150323_004390	{memo X}Look closely at the magic field/LOOK_SIT/3
 QUEST_LV_0200_20150323_004391	{memo X}Notict/!/Bring the monsters near the magic field!#5
 QUEST_LV_0200_20150323_004392	What is the food
 QUEST_LV_0200_20150323_004393	{memo X}About what happened to Venews
-QUEST_LV_0200_20150323_004394	The unexpected discovery(1)
+QUEST_LV_0200_20150323_004394	$Unexpected Discovery(1)
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Unexpected Discovery (1)

**(add a space)**

File: QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -4391,23 +4391,23 @@ QUEST_LV_0200_20150323_004390	{memo X}Look closely at the magic field/LOOK_SIT/3
...
 QUEST_LV_0200_20150323_004395	{memo X}Looking closely at it/LOOK_SIT/1/TRACK_BEFORE/None
 QUEST_LV_0200_20150323_004396	{memo X}Notice/!/This is the part that belongs to the head of the statue of the Goddess! Break it!#5
 QUEST_LV_0200_20150323_004397	{memo X}Breaking the statue/SITGROPE/2/TRACK_BEFORE/None
-QUEST_LV_0200_20150323_004398	The unexpected discovery(2)
+QUEST_LV_0200_20150323_004398	$Unexpected Discovery(2)
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Unexpected Discovery (2)

**(add a space)**

File: QUEST_LV_0200.tsv [hunk 5]
```diff
@@ -4391,23 +4391,23 @@ QUEST_LV_0200_20150323_004390	{memo X}Look closely at the magic field/LOOK_SIT/3
...
 QUEST_LV_0200_20150323_004399	{memo X}Looking closely inside the alcohol container/LOOK/2/TRACK_BEFORE/SSN_HATE_AROUND
-QUEST_LV_0200_20150323_004400	The unexpected discovery(3)
+QUEST_LV_0200_20150323_004400	$Unexpected Discovery(3)
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Unexpected Discovery (3)

**(add a space)**

File: QUEST_LV_0200.tsv [hunk 6]
```diff
@@ -4391,23 +4391,23 @@ QUEST_LV_0200_20150323_004390	{memo X}Look closely at the magic field/LOOK_SIT/3
...
 QUEST_LV_0200_20150323_004401	{memo X}Looking closely inside the alcohol container/LOOK/2/TRACK_BEFORE/None
-QUEST_LV_0200_20150323_004402	The unexpected discovery(4)
+QUEST_LV_0200_20150323_004402	$Unexpected Discovery(4)
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Unexpected Discovery (4)

**(add a space)**

File: QUEST_LV_0200.tsv [hunk 7]
```diff
@@ -4391,23 +4391,23 @@ QUEST_LV_0200_20150323_004390	{memo X}Look closely at the magic field/LOOK_SIT/3
...
 QUEST_LV_0200_20150323_004403	{memo X}Looking closely at something which resembles wings/LOOK/3/TRACK_BEFORE/None
-QUEST_LV_0200_20150323_004404	The irregular stone statue
+QUEST_LV_0200_20150323_004404	$The Irregular Stone Statue
 QUEST_LV_0200_20150323_004405	{memo X}About Yoana
-QUEST_LV_0200_20150323_004406	You found that it was the blessing
+QUEST_LV_0200_20150323_004406	$A Blessing After All
 QUEST_LV_0200_20150323_004407	{memo X}About the work you are doing at the moment
-QUEST_LV_0200_20150323_004408	The positive evidence(1)
+QUEST_LV_0200_20150323_004408	$Positive Evidence(1)
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Positive Evidence (1)

**(add a space)**

File: QUEST_LV_0200.tsv [hunk 8]
```diff
@@ -4391,23 +4391,23 @@ QUEST_LV_0200_20150323_004390	{memo X}Look closely at the magic field/LOOK_SIT/3
...
 QUEST_LV_0200_20150323_004409	{memo X}About the aqueduct region
-QUEST_LV_0200_20150323_004410	The positive evidence(2)
+QUEST_LV_0200_20150323_004410	$Positive Evidence(2)
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Positive Evidence (2)

**(add a space)**

File: QUEST_LV_0200.tsv [hunk 9]
```diff
@@ -6338,10 +6338,10 @@ QUEST_LV_0200_20150406_006341	{memo X}Notice/!/You should collect the magical po
 QUEST_LV_0200_20150406_006342	I will protect the storage
 QUEST_LV_0200_20150406_006343	About the relationship between the power generator and the grains
 QUEST_LV_0200_20150406_006344	Forgotten magic field(1)
-QUEST_LV_0200_20150406_006345	{memo X}Notice/!/Go back to Rimas and report him about the magic field.#5
+QUEST_LV_0200_20150406_006345	{memo X}Notice/!/Go back to Limas and report him about the magic field.#5
 QUEST_LV_0200_20150406_006346	{memo X}Sprinkling murky purifying liquid/STANDSPINKLE/1/TRACK_BEFORE/None
 QUEST_LV_0200_20150406_006347	{memo X}Notice/!/The magic field wants the monsters strongly!#5
-QUEST_LV_0200_20150406_006348	About menticen
+QUEST_LV_0200_20150406_006348	$About Mentiken
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Mentiken -> Manticen

**Comment by Sunaries** [comment]
I noticed you didn't include the title of Limas. 

e.g. "Talk to **Magician/Wizard** Limas".

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**